src/main/resources/assets/js/application/recruiter/controllers/tdprWeekTableController.js

```diff
@@ -1,13 +1,22 @@
 define(['angular', 'application/recruiter/tdprRecruiterModule', 'application/recruiter/services/tdprRecruiterSlotsService'
 ], function (angular, tdprRecruiterModule) {
-    tdprRecruiterModule.controller("tdprWeekTableController", function ($scope, tdprPersonsService, tdprDateService, persons, slotsTimes, JobProfileEnum, Notification, tdprRecruiterSlotsService, AvailabilityEnum, dateFilter) {
+    tdprRecruiterModule.controller("tdprWeekTableController", function ($scope, tdprPersonsService, tdprDateService, persons, slotsTimes,
+                                                                        JobProfileEnum, Notification, tdprRecruiterSlotsService, AvailabilityEnum, dateFilter, $filter) {
 
         $scope.JobProfileEnum = JobProfileEnum;
         $scope.currentJobProfile = JobProfileEnum.dev;
 
         $scope.days = tdprDateService.getCurrentWeek();
         $scope.slotsTimes = slotsTimes;
+        $scope.selectSlotsTimes = slotsTimes;
         $scope.persons = persons;
+        $scope.endTime = slotsTimes[slotsTimes.length - 1].endTime;
+        $scope.startTime = slotsTimes[0].startTime;
```

Comment:
Are we confident that slotTimes array will always contain at least 2 elements? If not some checks should be implemented.